cg/store/crud/create.py

```diff
@@ -409,10 +409,11 @@ def add_sample_lane_sequencing_metrics(
 
     def add_order(self, order_data: OrderIn):
         customer: Customer = self.get_customer_by_internal_id(order_data.customer)
+        workflow: str = order_data.samples[0].data_analysis
```

Comment:
Can we be sure that samples exist and that the data analysis is set on them?

Reply:
I interpret this as us having minimum one sample: 
<img width="915" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/72c44ccf-33bd-4035-bf7c-4b0bf1963866">

and data_analysis is not nullable in the basemodel 
<img width="496" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/0fd97d9b-54c3-44fb-8a21-b3b9aa5f535f">


Maybe @henrikstranneheim can confirm that we can not have an order without samples, and that we can not have samples without data_analysis set?

Reply:
We can always assume that all orders should have 1 sample, and that sample should always have a workflow. What I do not know is whether or not we validate this. But we probably do 👍

Reply:
Yeah, I have seen this validation at least in the new order portal. And if not, I would guess we would want to throw an error.